.flake8

```diff
@@ -12,9 +12,11 @@ max-complexity = 10
 max-cognitive-complexity = 10
 per-file-ignores =
     # No docs and annotation required for tests
+    /home/mukha/PycharmProjects/epam_python/hm_1/tasks/task_04_02.py: S310
```

Comment:
Why do you need this line?

Reply:
I add it because when github checks files crashed by lint with flake8, but if I set it up to local it doesn't exist and always okey.

Reply:
I don't sure if you wrote what you mean.  This looks like the local path on your machine. 

If you need something to work on different machines, would you use a relative or absolute path?